mechs/switches/index.rst

```diff
@@ -16,11 +16,17 @@ MPF's *switch* device represents a switch in a pinball machine. This device is
 used for switches, including cabinet buttons, rollovers, targets, optos, trough
 switches, DIP switches, etc.
 
-There are two general switch types in pinball machines (read those for details):
+There are two switch types most commonly seen in pinball machines (read those for details):
 
 * :doc:`Mechanical switches <mechanical_switches>`
 * :doc:`Optical switches <optos>`
 
+And an additional two types used in a handful of machines (read those for details):
+
+* :doc: 'Proximity switches <Proxy_switches>'
```

Comment:
Almost. The space after `:doc: ` needs to go. I'm not sure if you can create new pages in the web gui though. I can do that for you if you want.

Reply:
ok, I think it's fixed.  If you can create those blank pages I can add pictures and some info on them, that would be great while I get up to speed on GitHub

Reply:
Just checked. You can click "create new file" or "upload file" here: https://github.com/Beckermi/mpf-docs/tree/patch-1/mechs/switches